cabot/cabotapp/views.py

```diff
@@ -197,6 +197,7 @@ class Meta:
             'max_queued_build_time',
             'use_activity_counter',
             'runbook',
+            'max_build_failures'
```

Comment:
Let's put this above the activity counter and the runbook (more consistent UI/make it more visible)